addons/binding/org.openhab.binding.russoundserial/src/main/java/org/openhab/binding/russoundserial/internal/RussoundSerialHandlerFactory.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v1.0 which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.russoundserial.internal;
+
+import com.google.common.collect.Sets;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.math.BigDecimal;
+import java.util.Set;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import com.thejholmes.russound.Russound;
+import com.thejholmes.russound.RussoundCommandSender;
+import com.thejholmes.russound.RussoundCommander;
+import com.thejholmes.russound.RussoundTranslator;
+import com.thejholmes.russound.Zone;
+import com.thejholmes.russound.serial.SerialCommandReceiver;
+import com.thejholmes.russound.serial.SerialCommandSender;
+import org.openhab.binding.russoundserial.handler.RussoundSerialBridgeHandler;
+import org.openhab.binding.russoundserial.handler.RussoundSerialBridgeHandler.ZoneListener;
+import org.openhab.binding.russoundserial.handler.RussoundSerialZoneHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.openhab.binding.russoundserial.RussoundSerialBindingConstants.SERIAL_BRIDGE;
+import static org.openhab.binding.russoundserial.RussoundSerialBindingConstants.ZONE_HANDLER;
+
+/**
+ * The {@link RussoundSerialHandlerFactory} is responsible for creating things and thing handlers.
+ *
+ * @author holmes - Initial contribution
+ */
+public class RussoundSerialHandlerFactory extends BaseThingHandlerFactory {
+  private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS =
```

Comment:
Please remove the use of `com.google.common.collect.Sets` and the dependency to this library from the `Manifest`.

Instead you can use:
```
new HashSet<>(Arrays.asList(new String[] { SERIAL_BRIDGE, ZONE_HANDLER }))
```